app/src/graphql/operations.graphql

```diff
@@ -39,24 +39,15 @@ subscription CurrentGame($joinCode: String!) {
     cards(
       where: {
         _or: [
-          {
-            _and: [
-              { game: { screen_cards: { _eq: true } } }
-              { is_allowed: { _eq: true } }
-            ]
-          }
-          {
-            _and: [
-              { game: { screen_cards: { _eq: false } } }
-              { is_allowed: { _eq: null } }
-            ]
-          }
+          { is_allowed: { _is_null: true } }
+          { is_allowed: { _eq: true } }
```

Comment:
Open to suggestions here; see https://github.com/avimoondra/fishbowl/pull/98#discussion_r453320138.

Reply:
this makes sense, will try with neq false as you suggested